pkg/subscription/cluster-lifecycle.go

```diff
@@ -21,6 +21,7 @@ func ClusterLifecycle(m *operatorsv1.MultiClusterHub, overrides map[string]strin
 			},
 			"global": map[string]interface{}{
 				"pullPolicy":      utils.GetImagePullPolicy(m),
+				"imagePullPolicy": utils.GetImagePullPolicy(m),
```

Comment:
Does this need `pullPolicy` and `imagePullPolicy`?

Reply:
Just imagePullPolicy for the cluster-lifecycle chart